### DISCUSSION TITLE
Output Festival si no hay pinguinos en sector

### DISCUSSION THREAD

**Discussion by JavierMoralesD** [IIC2133-PUC/2024-2#81] [Tarea 1] [answered]
Hola. Quería saber que valores debemos dar como `{median_happiness}`, ` {happiest_penguin_id}`, y `{saddest_penguin_id}`, en el caso de que en el sector en específico no hayan pinguinos. ¿O podemos asumir que siempre habrá a lo menos un pinguino?
Aprovecho para preguntar si podemos también hacer ese supuesto para el evento _LEAVE_ (que hay al menos un pinguino en el sector con menor mediana).

**Comment by Segismundios** [comment, marked as answer]
Holaa @JavierMoralesD, puedes asumir que en todos los sectores siempre habrá al menos 1 pingüino, tanto para LEAVE como para FESTIVAL. 
Saludos!
